browser/js/login/inventory/inventory.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+app.config(function($stateProvider) {
+    $stateProvider.state("inventory", {
+	url: "/inventory",
+	controller: "InventoryCtrl",
+	templateUrl: "js/login/inventory/inventory.html"
+    });
+});
+
+app.controller("InventoryCtrl", function($scope, CurrentFactory, ItemFactory) {
+    $scope.currentUser = CurrentFactory.cloneCurrentUser();
```

Comment:
you can just do $scope.currentUser = CurrentFactory.current.user instead of cloning the object